src/jllvm/vm/GarbageCollector.cpp

```diff
@@ -389,7 +392,7 @@ void* jllvm::GarbageCollector::allocate(std::size_t size)
 
     void* result = m_bumpPtr;
     m_bumpPtr += size;
-    std::memset(result, 0, size);
```

Comment:
Note I removed this memset as it is redundant. The allocation space is zeroed out on constuction and garbage collection (see Line 306)